string subject
---
bool onList
bool roboticsCourse
int32 credits
string subjectName

